### PR TITLE
AC-487: Client continuously force stop and restarts

### PR DESCRIPTION
**Pull request by HelioStrike**
# AC-487: Client continuously force stop and restarts

## Description of what I changed

Added provider tag and models in AndroidManifest.xml .
Adding provider tags fixed the SecurityException, but now classes extending Model like Link, FormResource, etc. weren't getting invoked, which was fixed by listing them out in AndroidManifest.xml .

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-487

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -41,6 +41,10 @@
         <meta-data
             android:name="AA_DB_VERSION"
             android:value="@integer/aadbversion" />
+        <meta-data
+            android:name="AA_MODELS"
+            android:value="org.openmrs.mobile.models.Link, org.openmrs.mobile.models.FormResource, org.openmrs.mobile.models.EncounterType,
+             org.openmrs.mobile.models.Encountercreate, org.openmrs.mobile.models.Obscreate" />
```
Comment (f4ww4z) [inline openmrs-client/src/main/AndroidManifest.xml]: @HelioStrike Did you make sure to include all models in here?

**Comment by f4ww4z** [comment]
Remember to hit 'Request Code Review' on JIRA when you've answered / made changes.

File: openmrs-client/src/main/AndroidManifest.xml (hunk 1 above)
Comment (HelioStrike) [reply openmrs-client/src/main/AndroidManifest.xml]: @f4ww4z Yup. I put in all the classes that I found on searching ' extends Model '

**Comment by HelioStrike** [comment]
Ah, I forgot! Its done now. :)

**Pull request merged**